src/content/getting-started/cloud-provider-accounts/azure/index.md

```diff
@@ -1,48 +1,42 @@
 ---
-linkTitle: Azure
-title: Prepare an Azure subscription to run Giant Swarm workload clusters
-description: This guide will walk you through all necessary steps to set up an Azure subscription with approriate Role definition and Service Principal for operating Giant Swarm workload clusters.
-weight: 20
-menu:
-  main:
-    identifier: gettingstarted-cloudprovider-azure
-    parent: gettingstarted-cloudprovider
+title: "Prepare an Azure subscription to run Giant Swarm workload clusters"
+description: "This guide will walk you through all necessary steps to set up an Azure subscription with approriate Role definition and Service Principal for operating Giant Swarm worload clusters."
+date: "2020-05-19"
+type: page
+weight: 100
+tags: ["tutorial"]
 user_questions:
   - How do I prepare my Azure subscription for use with Giant Swarm?
   - What do I need to configure in Azure in order to run Giant Swarm clusters?
-aliases:
-  - /guides/prepare-azure-subscription-for-tenant-clusters/
-  - /guides/prepare-azure-subscription/
-owner:
-  - https://github.com/orgs/giantswarm/teams/team-celestial
 ---
 
 # Prepare an Azure subscription to run Giant Swarm workload clusters
 
-In a Giant Swarm installation the workload clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription from the management cluster. This gives greater flexibility depending on the requirements and the use case. For example, it allows the management cluster to be running in one Azure subscription, while workload clusters operate in different Azure subscriptions, depending on the customer entities using them.
+In a Giant Swarm installation the workload clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription from the control plane. This gives greater flexibility depending on the requirements and the use case. For example, it allows the control plane to be running in one Azure subscription, while workload clusters operate in different Azure subscriptions, depending on the customer entities using them.
```

Comment:
Why these changes? I think we intentionally moved from `control plane` -> `management cluster` because `control plane` term is ambiguous between MC and WC?

See https://github.com/giantswarm/giantswarm/issues/14961

Reply:
This is a rebase error, thanks for catching.
Let me fix it